setup.cfg

```diff
@@ -15,7 +15,7 @@ package_dir =
     = src
 packages = find:
 install_requires =
-    aiokatcp>=1.9.0
+    aiokatcp>=2.0.0
```

Comment:
I'd revert this one. The package should still work fine with 1.9.0, and if someone wants to install it that way we shouldn't stop them; we just choose to deploy with 2.0.0 ourselves.

Reply:
Ah, noted!